lib/monitoring/monitoringClientInterface.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const assert = require('assert')
+const client = require('prom-client');
+
+/**
+ * Wrapper for Prometheus client - https://github.com/siimon/prom-client
+ */
+
+var references = {};
```

Comment:
No need to keep another layer of references, `prom-client` has `getSingleMetric` on a `Registry`.

Reply:
Makes sense. I have removed references.